muskathlon/__manifest__.py

```diff
@@ -62,7 +62,7 @@
         'templates/muskathlon_order_material.xml',
         'templates/assets.xml'
     ],
-    'depends': ['website_event_compassion', 'survey'],
+    'depends': ['website_event_compassion', 'survey', 'partner_compassion'],
```

Comment:
website_event_compassion is already depending on partner_compassion (through the following chain dependency: website_event_compassion -> partner_communication_switzerland -> report_compassion -> sponsorship_switzerland -> partner_compassion.

Thus, you don't need to add this direct dependency.